extension/postgres_scanner/src/postgres_catalog.cpp

```diff
@@ -0,0 +1,45 @@
+#include "postgres_catalog.h"
+
+#include "common/exception/binder.h"
+#include "postgres_storage.h"
+
+namespace kuzu {
+namespace postgres_scanner {
+
+void PostgresCatalogContent::init(
+    const std::string& dbPath, const std::string& /*catalogName*/, main::ClientContext* context) {
+    duckdb_scanner::DuckDBCatalogContent::init(dbPath, DEFAULT_CATALOG_NAME, context);
+}
+
+std::string PostgresCatalogContent::getDefaultSchemaName() const {
+    return DEFAULT_SCHEMA_NAME;
+}
+
+std::unique_ptr<binder::BoundCreateTableInfo> PostgresCatalogContent::bindCreateTableInfo(
+    duckdb::Connection& con, const std::string& tableName, const std::string& dbPath,
+    const std::string& /*catalogName*/) {
+    std::vector<binder::PropertyInfo> propertyInfos;
+    if (!bindPropertyInfos(con, tableName, DEFAULT_CATALOG_NAME, propertyInfos)) {
+        return nullptr;
+    }
+    return std::make_unique<binder::BoundCreateTableInfo>(common::TableType::FOREIGN, tableName,
```

Comment:
Normally I avoid excessive long line of code. So I would do
```
auto extraInfo = std::make_unique
return std::make_unique<BoundCreateTableInfo>(extraInfo).
```

Other notes
- Any reason you are using unique_ptr for `BoundCreateTableInfo`, I think we use object in the core.
- I'm assuming hard code "memory" is fine.

Reply:
I think the BoundCreateTableInfo requires a pointer to BoundCreateTableinfo